pilot/pkg/proxy/envoy/v2/debounced_channel_test.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2_test
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	v2 "istio.io/istio/pilot/pkg/proxy/envoy/v2"
+)
+
+func newBoolDebouncedChannel(debounceAfter time.Duration, debounceMax time.Duration, initialAccumulator bool,
```

Comment:
I don't think it makes sense to test bool channel when all of our code uses UpdateReq

Reply:
The tests are easier and more focused on the features provided by `DebouncedChannel`